compiler/src/dmd/statementsem.d

```diff
@@ -5054,3 +5054,241 @@ bool checkLabel(GotoStatement gs)
     }
     return false;
 }
+
+/*****************************************
+* Turn an aggregate which is an array into an expression tuple
+* of its elements. I.e., lower
+*     static foreach (x; [1, 2, 3, 4]) { ... }
+* to
+*     static foreach (x; AliasSeq!(1, 2, 3, 4)) { ... }
+*/
+    extern(D) void lowerArrayAggregate(StaticForeach stf, Scope* sc)
+    {
```

Comment:
Can remove four spaces of indentation here.
```suggestion
*/
extern(D) void lowerArrayAggregate(StaticForeach stf, Scope* sc)
{
```
Same applies for the rest of the moved code.